### PR TITLE
Complete website

### PR DESCRIPTION
**Pull request by carlos-osorio-developer**
## Add functionality for:

- Navbar header.
- Footer.
- Contact section.
- Script to change display CSS property according to navbar navigation links.

### CODE REVIEW

File: assets/js/sections.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const propSections = {
```
Comment (ZawadiSumba66) [inline assets/js/sections.js:1]: - [x] Your project is currently lacking the current date display. For setting the time you are supposed to use the luxon library as suggested in the requirements. Please fix this.

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
#### Hi @carlos-osorio-developer  and @pountzas  ✋ , 

Good job so far!

### HIGHLIGHTS 🚀 🚀 

 - Great UI design 👍
 - Well-organized code.
- Descriptive README file.

However, there are some issues that you still need to work on to go to the next step but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/js/sections.js (hunk 1 above)
Comment (pountzas) [reply assets/js/sections.js:1]: Yes we forgot it thanks.

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @carlos-osorio-developer,  

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**